.rubocop_todo.yml

```diff
@@ -26,9 +26,9 @@ Metrics/AbcSize:
 # Offense count: 2
 # Configuration parameters: IgnoredMethods.
 Metrics/CyclomaticComplexity:
-  Max: 9
+  Max: 10
```

Comment:
This rule and `PerceivedComplexity` were failing on `connection` with the newly added middleware.

Reply:
Generally I just run `rubocop -a ; rubocop --auto-gen-config`.